acceptance/framework/config/config.go

```diff
@@ -52,6 +52,9 @@ type TestConfig struct {
 	ConsulVersion    *version.Version
 	EnvoyImage       string
 
+	VaultHelmChartVersion string
+	VaultServerVersion    *version.Version
```

Comment:
I *think* that `VaultServerVersion` can be treated as a string, iirc the reason `ConsulVersion` used type `version.Version` was so we could use a library to do the version comparisons for feature flagging: https://github.com/hashicorp/consul-k8s/blob/main/acceptance/tests/vault/vault_partitions_test.go#L34-L38

I don't think that the VaultServerVersion is going to be something that the acceptance tests would need to feature flag though, so it's probably okay to make this a string throughout!

Reply:
I suggested switching to the type as the other code depended on it being that type (or at least not a string). You could also change those places to support a string if you'd rather go that way. Mainly the `nil` check and assignment below... (I'll comment there).